packages/sanity/src/components/collapseMenu/CollapseMenuButton.tsx

```diff
@@ -1,20 +1,18 @@
 import {Button, ButtonProps, TooltipProps} from '@sanity/ui'
 import React, {forwardRef} from 'react'
 
-export interface CommonProps extends Omit<ButtonProps, 'text' | 'icon' | 'iconRight'> {
+export type CommonProps = {
```

Comment:
This is rewritten to reuse `ButtonProps` parts instead of re-defining them. `@sanity/ui` have, for example, `React.ElementType | React.ReactNode` as the type for `icon`